src/main/java/de/zalando/aruha/nakadi/controller/EventPublishingController.java

```diff
@@ -75,6 +78,10 @@ private ResponseEntity postEventInternal(String eventTypeName, String eventsAsSt
         }
     }
 
+    private ThrowableProblem crateProblem(JSONException e) {
+        return Problem.valueOf(Response.Status.BAD_REQUEST, e.getMessage());
```

Comment:
This is JSONException, I think in this case it's ok to expose the message to user, because this is exactly what he wants to see: what is wrong with the json he provided.